src/components/Application/Sections/SectionsList.tsx

```diff
@@ -8,11 +8,22 @@ interface SectionsListProps {
 
 const SectionsList: React.FC<SectionsListProps> = ({ sections }) => {
   return (
-    <List celled relaxed="very">
-      {Object.entries(sections).map(([sectionCode, { title }]) => (
-        <List.Item key={`list-item-${sectionCode}`} icon="circle outline" header={title} />
-      ))}
-    </List>
+    <List
+      celled
+      relaxed="very"
+      items={Object.entries(sections).map(([sectionCode, { title }]) => ({
+        key: `list-item-${sectionCode}`,
+        content: (
+          <div className="line-vertical-box">
+            <div className="centered-flex-box-row">
```

Comment:
See what I mean -- these are already used in a nested/heirarchical way, so if `line-vertical-box` has some general properties and `centered-flex-box-row` adds some more specific properties to the items contained within, it'll just work -- no need to define them together in the CSS.